distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -265,6 +278,9 @@ public void beforeExam() throws Exception {
 
     @Before
     public void setup() throws Exception {
+
+        getCatalogBundle().setDownloadRetryDelayInSeconds(1);
```

Comment:
@figliold Will this adversely impact other tests that aren't part of @cmthom10's series? Should we extract new `setup()` code into a private method called exclusively before her tests?